documentation/docs/steps/apiKeyValueMapUpload.md

```diff
@@ -0,0 +1,47 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+With this step you can store one or more key value pairs of data stored in a group called a map or key value map.<br />
```

Comment:
Add empty line below, instead of `<br />`
```suggestion
With this step you can store one or more key value pairs of data stored in a group called a map or key value map.

```